jbpm-persistence/jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/processinstance/JPASignalManager.java

```diff
@@ -16,12 +16,14 @@
 
 package org.jbpm.persistence.processinstance;
```

Comment:
@mariofusco  do you mind to remove this file change. I will treat the batch system in a different PR.

Reply:
yes. it should

Reply:
@mariofusco